dynatrace-aws-logs.sh

```diff
@@ -150,6 +147,65 @@ arguments:
   aws cloudformation describe-stacks --stack-name "$STACK_NAME" --query "Stacks[0].Outputs"
   ;;
 
+"delete")
+
+  function print_help_delete {
+    printf "
+usage: dynatrace-aws-logs.sh delete [--stack-name STACK_NAME]
+
+arguments:
+  -h, --help            show this help message and exit
+  --stack-name STACK_NAME
+                        Optional. The name for the CloudFormation stack in which the resources will be deployed. This defaults to \"$DEFAULT_STACK_NAME\"
```

Comment:
"will be deployed" not correct in this option :D

Reply:
Ahh.

Reply:
Fixed.